flink-clients/src/test/java/org/apache/flink/client/ClientUtilsTest.java

```diff
@@ -26,8 +26,15 @@
 import org.apache.flink.util.SerializedThrowable;
 import org.apache.flink.util.TestLogger;
 
-import org.junit.Assert;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Assertions;
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.Timeout;
+import static org.hamcrest.MatcherAssert.assertThat;
+import org.junit.jupiter.api.Assertions;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import org.hamcrest.MatcherAssert;
+import static org.junit.jupiter.api.Assertions.assertTrue;
```

Comment:
are all these imports actually necessary or was it some quick-and-dirty approach?

Reply:
Yes, I was experimenting with changing most code with some generic replaces and do a final cleanup with spotless. Note that this PR is still very WIP - I just needed more AZP power.